sql/_init.sql

```diff
@@ -66,20 +66,19 @@ OUTPUTS:
 */
 
 DROP TABLE IF EXISTS _INIT_devdb;
-WITH combined AS (
+SELECT *
+INTO _INIT_devdb
+FROM (
 	SELECT *,
 		'bis' AS datasource
 	FROM _INIT_BIS_devdb
 	UNION
-	SELECT *,
+	SELECT 
+		ROW_NUMBER() OVER(ORDER BY job_number) + (SELECT MAX(uid) FROM _INIT_BIS_devdb) as uid,
```

Comment:
Are we not using the hash uid created in now/_init.sql?

Reply:
no, the hash uid is a text field but the bis uid is a integer field. I'm working on a fix rn